infra/nodes/akka-node.json.template

```diff
@@ -0,0 +1,22 @@
+{
+    "chef_environment": "_default", 
+    "ipaddress-internal": "INTERNAL_IP", 
+    "ipaddress": "EXTERNAL_IP", 
+    "name": "NAME", 
+    
+    "akka": {
+      "seed_nodes": AKKA_SEED_NODES
+    },
+    
+    "java": {
+      "jdk_version": 8,
+      "oracle": {
+        "accept_oracle_download_terms": true
+      },
+      "url": "http://download.oracle.com/otn-pub/java/jdk/8u112-b15/server-jre-8u112-linux-x64.tar.gz"
+    },
+    "run_list": [
+        "role[benchmark]",
+        "role[gce-tools]"
+    ]
+}
```

Comment:
see this; if we want to spin up a lot of nodes with weird things, we would edit this file instead of the node files